src/app/components/Item/ItemHoldings.jsx

```diff
@@ -138,7 +138,9 @@ class ItemHoldings extends React.Component {
 
     return (
       <div id="item-holdings" className="item-holdings">
-        <h2>{this.props.title}</h2>
+        <dt className="list-multi-control">
```

Comment:
This "Availability" is a little strange -- I think this is in flux with the design-toolkit right now -- because the SASS requires the h3 to be wrapped in the dt with this class name in order to be styled...